common/deposits/zz_evolved_pdhab_deposits.txt

```diff
@@ -3,7 +3,7 @@
 d_ggstorm15 = {
 	is_for_colonizable = yes
 	use_for_min_max_adjustments = yes
-	category = deposit_cat_blockers
+	category = deposit_cat_rare_blocker

```

Comment:
Are the rare category blockers change needed, cost wise only the resource category should have the desired effect. In rest looks good!